public/app/features/org/NewOrgPage.tsx

```diff
@@ -37,8 +38,9 @@ interface CreateOrgFormDTO {
 }
 
 export const NewOrgPage: FC<Props> = ({ navModel, createOrganization }) => {
-  const createOrg = (newOrg: { name: string }) => {
-    createOrganization(newOrg);
+  const createOrg = async (newOrg: { name: string }) => {
+    await createOrganization(newOrg);
```

Comment:
Curious how this works. When calling a Thunk you should always wrap the call in `dispatch`. So this would effectively be

```ts
import { useDispatch } from 'react-redux';
...
const dispatch = useDispatch();
const createOrg = async (newOrg: { name: string }) => {
    await dispatch(createOrganization(newOrg));
    window.location.href = getConfig().appSubUrl + '/org'; 
};
```